src/rpdk/core/invoke.py

```diff
@@ -47,7 +47,11 @@ def invoke(args):
     try:
         while _needs_reinvocation(args.max_reinvoke, current_invocation):
             print("=== Handler input ===")
-            print(json.dumps({**payload, "credentials": "<redacted>"}, indent=2))
+
+            payload_copy = payload.copy()
+            payload_copy["requestData"]["callerCredentials"] = "<redacted>"
```

Comment:
I think we have a  list of loggable properties instead of removing some. Can you use the same approach ?

Reply:
Could you guys send me a link to the loggable properties.

Reply:
https://github.com/aws-cloudformation/cloudformation-cli/blob/master/src/rpdk/core/contract/resource_client.py#L588-L600

Reply:
invoke is calling: https://github.com/aws-cloudformation/cloudformation-cli/blob/master/src/rpdk/core/contract/resource_client.py#L563 which in turn calls the above method. Why do we want to construct the model again?

Reply:
It's like allow-list vs disallow-list. Allow-list is safer that if any new property introduced , it won't show up in the loggable propeties unless it added.